clang/lib/CIR/CodeGen/CIRGenBuiltin.cpp

```diff
@@ -1272,9 +1272,6 @@ RValue CIRGenFunction::emitBuiltinExpr(const GlobalDecl &gd, unsigned builtinID,
     llvm_unreachable("Bad evaluation kind in EmitBuiltinExpr");
   }
 
-  cgm.errorNYI(e->getSourceRange(),
```

Comment:
This check is now handled in `CIRGenBuiltinX86`, so we can return empty values without triggering the NYI error

Reply:
I don't think that's right. That will only report the error for missing X86 builtins. This was a backstop for everything else.

Reply:
I caught missing builtins for other targets in `emitTargetArchBuiltinExpr`